src/java/fr/paris/lutece/plugins/myfiles/service/storage/impl/MinioFileStorage.java

```diff
@@ -112,7 +118,9 @@ public MinioFileStorage( )
                     myFile.setName( item.objectName( ) );
                     ObjectStat stat = _client.statObject( strBucketName, item.objectName( ) );
                     myFile.setContentType( stat.contentType( ) );
-                    String strUrl = _client.presignedGetObject( strBucketName, item.objectName( ), 60 * 60 * 24 );
+                    myFile.setSize( stat.length(  ) );
```

Comment:
Yes, it should be as it's not related to the commit message. Thus, size isn't currently displayed in home view. I made a new commit to add this.